Resources/Prototypes/Stray/Entities/Clothing/Head/hardsuit-helmet.yml

```
@@ -23,3 +23,20 @@
+  name: шлем научного скафандра #science hardsuit helmet
+  description: Легкий жесткий шлем, не ограничивающий движения головы. #Lightweight hardsuit helmet that doesn't restrict your head movements.
+  components:
+  - type: Sprite
+    sprite: Stray/Clothing/Head/Hardsuits/sciencehead.rsi
+  - type: Clothing
+    sprite: Stray/Clothing/Head/Hardsuits/sciencehead.rsi
+  - type: PointLight
+    color: "#a4fef4"
+  - type: PressureProtection
+    highPressureMultiplier: 0.60
+    lowPressureMultiplier: 1000
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)